kong/plugins/statsd/schema.lua

```diff
@@ -1,24 +1,163 @@
 local metrics = {
-  "request_count",
-  "latency",
-  "request_size",
-  "status_count",
-  "response_size",
-  "unique_users",
-  "request_per_user",
-  "upstream_latency"
+  ["request_count"]         = true,
+  ["latency"]               = true,
+  ["request_size"]          = true,
+  ["status_count"]          = true,
+  ["response_size"]         = true,
+  ["unique_users"]          = true,
+  ["request_per_user"]      = true,
+  ["upstream_latency"]      = true,
+  ["kong_latency"]          = true,
+  ["status_count_per_user"] = true,
```

Comment:
fyi: when the `key` is a valid Lua identifier name, you can omit the quotes and brackets:
```lua
local metrics = {
  request_count         = true,
  latency               = true,
```
would do the job
(no need to change)